Algorithm.CSharp/BasicPythonIntegrationTemplateAlgorithm.cs

```diff
@@ -0,0 +1,56 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Data;
+using Python.Runtime;
+namespace QuantConnect.Algorithm.CSharp
+{
+    public class BasicPythonIntegrationTemplateAlgorithm : QCAlgorithm
+    {
+
+        public override void Initialize()
+        {
+            SetStartDate(2018, 11, 10);  //Set Start Date
+            SetEndDate(2018, 12, 1);
```

Comment:
For consistency, we should keep the same date range in both algorithms.

Reply:
I made the change in the python algo, they'll be the same now